### ISSUE TITLE
YouTube doesn't play after update to 5.15

### ISSUE THREAD

**Issue by dmik** [bitwiseworks/qtwebengine-chromium-os2#45]
It starts playing for a few seconds, then playback stops. Note that vimeo and rutube videos play just fine so it must be something specific to the codec YouTube uses.

**Comment by dryeo** [comment]
Right click on the video and choose "Stats for nerds". It will show the 
codecs used, likely Opus for audio.
If you can disable Opus, Youtube should fall back on a different codec, 
likely AAC if that is enabled.

**Comment by dmik** [comment]
Yes, it's Opus for audio but it is Opus in 5.13.1 as well and there it works just fine. So it must be some regression in Chromium itself.

This is from 5.15:

![image](https://user-images.githubusercontent.com/900726/127694155-8b2740c5-8c11-4732-8c07-d3867760ef1c.png)

This is from 5.13:

![image](https://user-images.githubusercontent.com/900726/127694729-ff9c747c-2dad-437a-89d2-ada7bfc85402.png)

**Comment by dryeo** [comment]
Seems to be an av1 (libdav1d) issue, vp9 videos play fine.
